### PR TITLE
Add support for getting license report from service

### PR DESCRIPTION
**Pull request by zozs**
Adds new command for generating license report.

### CODE REVIEW

File: src/Command/LicenseReportCommand.php [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+    public const ARGUMENT_PROFILE = 'profile';
+    public const ARGUMENT_UPLOAD_ID = 'upload-id';
+    public const OPTION_FORMAT = 'format';
+    public const OPTION_OUTPUT_FILE = 'output';
+    public const OPTION_SNIPPETS = 'snippets';
+
+    /**
+     * @var HttpClientInterface
+     */
+    private $debrickedClient;
```
Comment (sweoggy) [inline src/Command/LicenseReportCommand.php:44]:
```suggestion
    private HttpClientInterface $debrickedClient;
```

File: src/Command/LicenseReportCommand.php [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+            ->addArgument(
+                self::ARGUMENT_UPLOAD_ID,
+                InputArgument::REQUIRED,
+                "Upload id you got from running $findAndUploadCommand",
+                null
+            )
+            ->addArgument(
+                self::ARGUMENT_PROFILE,
+                InputArgument::REQUIRED,
+                'The license risk profile you wish to use for this scan',
```
Comment (sweoggy) [inline src/Command/LicenseReportCommand.php:83]: Can we list the possible values for this argument?

File: src/Command/LicenseReportCommand.php (hunk 1 above)
Comment (zozs) [reply src/Command/LicenseReportCommand.php]: Fixed :)

File: src/Command/LicenseReportCommand.php (hunk 2 above)
Comment (zozs) [reply src/Command/LicenseReportCommand.php]: Nice catch. Yes we can, and now we do :).

**Comment by sweoggy** [comment]
Good job!

**Pull request merged**